src/elevator/ElevatorSystem.java

```diff
@@ -1,58 +1,77 @@
 package elevator;
 
-import java.util.ArrayList;
 import java.util.LinkedList;
-import java.util.List;
 import java.util.Queue;
 
 import common.Message;
-import common.Requestable;
+import common.MessageType;
 import scheduler.Scheduler;
 
+/**
+ * System for managing elevators and receive messages from scheduler
+ * 
+ * @author Derek Shao, Souheil Yazji
+ *
+ */
 public class ElevatorSystem implements Runnable {
-	
-	private Requestable scheduler; 
-	private List<Elevator> elevators;
-	private Queue<Message> requests;
-	
+
+	private Scheduler scheduler;
+	private Elevator ele1;
```

Comment:
Why only hardcode 1 elevator?

Reply:
Just for iteration 1
It wouldn't be too big of a change for multiple elevators